src/Shared/AWS/AWSSemanticConventions.v1.27.0.cs

```diff
@@ -0,0 +1,105 @@
+    /// </remarks>
+    private class AWSSemanticConventions_v1_27_0 : AWSSemanticConventionsBase
+    {
+        // CLOUD Attributes
+        public override string AttributeCloudAccountID => "cloud.account.id";
+        public override string AttributeCloudAvailabilityZone => "cloud.availability_zone";
+        public override string AttributeCloudPlatform => "cloud.platform";
+        public override string AttributeCloudProvider => "cloud.provider";
+        public override string AttributeCloudRegion => "cloud.region";
+        public override string AttributeCloudResourceId => "cloud.resource_id";
+        public override string CloudPlatformValuesAwsEc2 => "aws_ec2";
+        public override string CloudPlatformValuesAwsEcs => "aws_ecs";
+        public override string CloudPlatformValuesAwsEks => "aws_eks";
+        public override string CloudPlatformValuesAwsElasticBeanstalk => "aws_elastic_beanstalk";
+        public override string CloudProviderValuesAws => "aws";
+
+        // CONTAINER Attributes
+        public override string AttributeContainerID => "container.id";
+
+        // DB Attributes
+        public override string AttributeDbSystem => "db.system";
+        public override string AttributeDynamoDb => "dynamodb";
+
+        // AWS Attributes
+        public override string AttributeEcsContainerArn => "aws.ecs.container.arn";
+        public override string AttributeEcsClusterArn => "aws.ecs.cluster.arn";
+        public override string AttributeEcsLaunchtype => "aws.ecs.launchtype";
+        public override string ValueEcsLaunchTypeEc2 => "ec2";
+        public override string ValueEcsLaunchTypeFargate => "fargate";
+        public override string AttributeEcsTaskArn => "aws.ecs.task.arn";
+        public override string AttributeEcsTaskFamily => "aws.ecs.task.family";
+        public override string AttributeEcsTaskRevision => "aws.ecs.task.revision";
+        public override string AttributeLogGroupNames => "aws.log.group.names";
+        public override string AttributeLogGroupArns => "aws.log.group.arns";
+        public override string AttributeLogStreamNames => "aws.log.stream.arns";
+        public override string AttributeLogStreamArns => "aws.log.stream.names";
+        public override string AttributeAWSDynamoTableName => "aws.table_name";
+        public override string AttributeAWSSQSQueueUrl => "aws.queue_url";
+        public override string AttributeAWSBedrockAgentId => "aws.bedrock.agent.id";
+        public override string AttributeAWSBedrockDataSourceId => "aws.bedrock.data_source.id";
+        public override string AttributeAWSBedrockGuardrailId => "aws.bedrock.guardrail.id";
+        public override string AttributeAWSBedrockKnowledgeBaseId => "aws.bedrock.knowledge_base.id";
+        public override string AttributeAWSBedrock => "aws_bedrock";
+
+        // FAAS Attributes
+        public override string AttributeFaasID => "cloud.resource_id";
```

Comment:
This was previously `"faas.id"` (matching the identifier's name). It seems this now always becomes cloud.resource_id. I don't know in which version faas.id was removed, was that already before 1.27? If so, I think since we are already going out of our way to offer different versions, we should also support the one that was previously used here.

Reply:
@Oberon00 - good catch. v1.27 is meant to track the current state of the OpenTelemeetry.*.AWS libraries - so I've reverted this back to `faas.id`.  v1.29 will upgrade to `cloud.resource_id`.

I rechecked the remaining faas attributes, but let me know if you see any other inconsistencies.